jest.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   collectCoverage: true,
   collectCoverageFrom: ['./src/**.ts'],
+  coveragePathIgnorePatterns: ['/node_modules/', 'index.ts'],
```

Comment:
The `jest` coverage reporter (i.e. `istanbul`) registers every exported function in `index.ts` as a distinct function, tanking our function coverage.